### ISSUE TITLE
[BUG]Skip ci does not take effect in non-English version

### ISSUE THREAD

**Issue by fisherzrj** [vivaxy/vscode-conventional-commits#257]
### Describe the bug

In the case of Chinese, when I choose to skip the ci workflow, there is no [skip ci] added to the commit information.

### To reproduce

1. Tick skip ci workflow
2. Select Yes when submitting a commit
3. View vscode commit message

### Expected behavior

[skip ci] appears after the description

### Screenshots

none

### Application logs
I think I've found where the problem is ,the judgment about skip ci in the src/lib/commit-message.ts file is as follows
```
107  if (ci === 'Yes') {
108    result += ' [skip ci]';
109  }
```
But the corresponding value in package.nls.zh-cn.json is indeed this
```
"extension.sources.prompt.ci.accept.label": "是",
```
"是" cannot be equal to "Yes", so I think we can modify this.
It is easier to change "是" to "Yes", because "Yes" is a widely used word and does not need translation

**Comment by yi-Xu-0100** [comment]
👋 @fisherzrj, thanks for the heads-up, I've fixed it in PR #258, and you can continue to work with the beta version. 😁

@vivaxy If there are no other issues, the new version will be released on Friday. 🙌

**Comment by yi-Xu-0100** [comment]
@vivaxy @fisherzrj Sorry for the later update, the new version 1.24.2 has been released. 🎉🎉🎉
